home.go

```diff
@@ -324,6 +331,17 @@ func (p *HomePage) Event(gtx layout.Context) interface{} {
 }
 
 func (p *HomePage) Start(stop <-chan struct{}) {
+	go func() {
+		for {
+			connectIconIdx = (connectIconIdx + 1) % len(connectIcons)
```

Comment:
Is this necesssary here when we already do it in `SpoolPage.Start` (where it actually gets drawn?)